src/lib.rs

```diff
@@ -255,131 +255,46 @@ impl Builder {
-            })
-            .count();
-
-        self.options
-            .new_type_alias
-            .get_items()
-            .iter()
-            .map(|item| {
-                output_vector.push("--new-type-alias".into());
-                output_vector.push(item.to_owned());
-            })
-            .count();
-
-        self.options
-            .new_type_alias_deref
-            .get_items()
-            .iter()
-            .map(|item| {
-                output_vector.push("--new-type-alias-deref".into());
-                output_vector.push(item.to_owned());
-            })
-            .count();
-
-        self.options
-            .blacklisted_types
-            .get_items()
-            .iter()
-            .map(|item| {
-                output_vector.push("--blacklist-type".into());
-                output_vector.push(item.to_owned());
-            })
-            .count();
-
-        self.options
-            .blacklisted_functions
-            .get_items()
-            .iter()
-            .map(|item| {
-                output_vector.push("--blacklist-function".into());
-                output_vector.push(item.to_owned());
-            })
-            .count();
+        let regex_sets = &[
+            (&self.options.bitfield_enums, "--bitfield-enum"),
+            (&self.options.newtype_enums, "--newtype-enum"),
+            (&self.options.rustified_enums, "--rustified-enum"),
+            (
+                &self.options.rustified_non_exhaustive_enums,
+                "--rustified-enum-non-exhaustive",
+            ),
+            (
+                &self.options.constified_enum_modules,
+                "--constified-enum-module",
+            ),
+            (&self.options.constified_enums, "--constified-enum"),
+            (&self.options.type_alias, "--type-alias"),
+            (&self.options.new_type_alias, "--new-type-alias"),
+            (&self.options.new_type_alias_deref, "--new-type-alias-deref"),
+            (&self.options.blacklisted_types, "--blacklist-type"),
+            (&self.options.blacklisted_functions, "--blacklist-function"),
+            (&self.options.blacklisted_items, "--blacklist-item"),
+            (&self.options.opaque_types, "--opaque-type"),
+            (&self.options.whitelisted_functions, "--whitelist-function"),
+            (&self.options.whitelisted_types, "--whitelist-type"),
+            (&self.options.whitelisted_vars, "--whitelist-var"),
+            (&self.options.no_partialeq_types, "--no-partialeq"),
+            (&self.options.no_copy_types, "--no-copy"),
+            (&self.options.no_hash_types, "--no-hash"),
+        ];
 
-        self.options
-            .blacklisted_items
-            .get_items()
-            .iter()
-            .map(|item| {
-                output_vector.push("--blacklist-item".into());
+        for (set, flag) in regex_sets {
+            for item in set.get_items() {
+                output_vector.push((*flag).to_owned());
```

Comment:
I assume the use of `(*flag).to_owned()` here, rather than `flag.to_owned()`, is to avoid the generic (slow) implementation of `ToString` for `&&str`.

I do not know whether this is worth a comment in the source code, or whether this is common knowledge to everyone.

Reply:
It's not about slowness here, it's about `ToOwned` for `&&str` returning an `&str`, not a `String` :-)

Reply:
Ah, yes -- I thought the use of `to_owned()` was just the idiom you preferred rather than `to_string()` here. Thanks for explaining.